spec/spec_helper.rb

```diff
@@ -60,8 +60,12 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-  # VCR cassetes config
 
+  config.after :all do
+    Timecop.return
+  end
```

Comment:
This should not be necessary. It is a big source of problems in some of our apps. Please use [safe mode](https://github.com/travisjeffery/timecop#timecopsafe_mode) instead.